formstest/src/main/java/org/odk/collect/formstest/InstancesRepositoryTest.java

```diff
@@ -281,6 +282,19 @@ public void softDelete_deletesInstanceDir() {
         assertThat(instanceDir.exists(), is(false));
     }
 
+    @Test
+    public void softDelete_clearsGeometryData() {
```

Comment:
I don't think we should use "soft delete" when talking about instances anywhere. There is only one kind of delete and it deletes everything other than metadata. Is that used anywhere else or is it just introduced here?

Reply:
You can also hard delete the Instance if it's not submitted though. I'm not sure if I can think of a better name for this but open to suggestions!